src/site/content/en/blog/performance-tool-score-diffs/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Performance Tooling Score Differences
+subhead: Why do I get different scores in Test My Site, PageSpeed Insights, and Lighthouse?
+authors:
+  - egsweeny
+date: 2019-06-12
+hero: hero.jpg
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+alt: Car speedometer.
+description: 
+  This post serves to explain the reasons that there are differences in performance scores between Test My Site (TMS), PageSpeed Insights (PSI), and Lighthouse. This document does not detail other differences in the tools, such as server run locations, network throttling presets, etc.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+[Load is not a single moment in time](https://developers.google.com/web/fundamentals/performance/user-centric-performance-metrics) — it's an experience that no one metric can fully capture. There are multiple moments during the load experience that can affect whether a user perceives it as "fast", and if you just focus on one you might miss bad experiences that happen during the rest of the time. **This is why we generally use multiple metrics blended together to create our performance scores;** but different tools use different metrics and different weighted blends. This can be frustrating when you're looking for a source of truth about how fast your website is.
```

Comment:
I was wondering if starting with an external link might tempt the user to immediately leave what you're trying to get them to read. I think later on you have a list which mentions this same point so you could use the link there instead?

Reply:
That's a great idea. Shall I go in and shift it?

Reply:
yep i'm all done :)